test/singlenode_test.go

```diff
@@ -711,48 +711,71 @@ func TestFireTrackBySdp(t *testing.T) {
 	_, finish := setupSingleNodeTest("TestFireTrackBySdp")
 	defer finish()
 
-	c1 := createRTCClient("c1", defaultServerPort, &testclient.Options{
-		ClientInfo: &livekit.ClientInfo{
-			Sdk: livekit.ClientInfo_JS,
+	var cases = []struct {
+		name   string
+		codecs []webrtc.RTPCodecCapability
+		pubSDK livekit.ClientInfo_SDK
+	}{
+		{
+			name: "js client could pub a/v tracks",
```

Comment:
does this work for simulcast publish also?

Reply:
simulcast tracks still needs to wait for rtp packet since there is no ssrc information in sdp